_includes/example.html

```diff
@@ -0,0 +1,23 @@
+{%- comment -%}
+Usage: {% include example.html content=markup %},
+where content is a capture with the HTML content
+id - null (default)
+class - "bd-example" (default)
+optional: hide_preview - disabled (default)
+optional: hide_markup - disabled (default)
+{%- endcomment -%}
+
+{%- assign preview_id = include.id -%}
+{%- assign preview_class = include.class | default: "bd-example" -%}
```

Comment:
I'm not sure this does what we want. We want to append a class we pass and always have `bd-example`. So maybe try using append with a space.